exporters/trace/jaeger/src/main/java/io/opencensus/exporter/trace/jaeger/JaegerTraceExporter.java

```diff
@@ -72,15 +73,15 @@ public static void createAndRegister(final String thriftEndpoint, final String s
    * Creates and registers the Jaeger Trace exporter to the OpenCensus library using the provided
    * HttpSender. Only one Jaeger exporter can be registered at any point.
    *
-   * @param httpSender the pre-configured HttpSender to use with the exporter
+   * @param sender the pre-configured ThriftSender to use with the exporter
    * @param serviceName the local service name of the process.
    * @throws IllegalStateException if a Jaeger exporter is already registered.
    * @since 0.17
    */
-  public static void createWithSender(final HttpSender httpSender, final String serviceName) {
+  public static void createWithSender(final ThriftSender sender, final String serviceName) {
```

Comment:
Will this be backwards-compatible? If not consider adding a separate `createWithSender` API that takes `ThriftSender`.

Reply:
HttpSender extends ThriftSender if i remember correctly; One should be able to just supply a HttpSender to the Method.

https://github.com/jaegertracing/jaeger-client-java/blob/20ee4e81749bbd0037389d04c32cfd296b3857d3/jaeger-thrift/src/main/java/io/jaegertracing/thrift/internal/senders/HttpSender.java#L34